inftrees.c

```
@@ -45,22 +46,53 @@ static inline void count_lengths(uint16_t *lens, int codes, uint16_t *count) {
+        lens += 2;
+    }
+
+    vector unsigned short sum_lo =
+        vec_add((vector unsigned short)vec_mergeh(zero, s1),
+                (vector unsigned short)vec_mergeh(zero, s2));
+
+    vector unsigned short sum_hi =
+        vec_add((vector unsigned short)vec_mergel(zero, s1),
+                (vector unsigned short)vec_mergel(zero, s2));
+
+    vec_st(sum_lo, 0, &count[0]);
```

Cppcheck (2.19.0)

[style] The function 'zng_memcmp_2' is never used.

(unusedFunction)